File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/factory/transportmethod/Bus.java
package com.bigboxcode.designpattern.factory.transportmethod;

public class Bus implements Transport {
    @Override
    public void start() {
        System.out.println("Bus started");
    }

    @Override
    public void stop() {
        System.out.println("Bus Stopped");
    }

    @Override
    public void repair() {
        System.out.println("Bus Repair");
    }
}

File: design-patterns/php/factory/transportmethod/Bus.php
<?php
// Bus.php

namespace BigBoxCode\DesignPattern\Factory\TransportMethod;

class Bus implements Transport {
    public function start() {
        echo "Bus started\n";
    }

    public function stop() {
        echo "Bus Stopped\n";
    }

    public function repair() {
        echo "Bus Repair\n";
    }
}